src/hooks/useEnabledExtensions.tsx

```diff
@@ -1,9 +1,11 @@
 import { Extension } from '@colony/colony-js';
 
+import { Colony } from '~types';
+
 import useExtensionsData from './useExtensionsData';
 
-const useEnabledExtensions = () => {
-  const { installedExtensionsData, loading } = useExtensionsData();
+const useEnabledExtensions = (colony: Colony | undefined) => {
```

Comment:
If we add `?`, this would mean that the param is optional no? We could call `useEnabledExtensions()` if that was the case, which is wrong because without a `colony` the hook will never return the data, as opposed to requiring a `colony` param that can be defined or not (We are assuming that eventually the `colony` will be defined, if not already).

Reply:
Fair enough, thanks 👍